### PR TITLE
[expo] Drop old SDKs

### PR DESCRIPTION
**Pull request by sjchmiela**
https://blog.expo.io/preview-upcoming-changes-for-expo-sdk-33-67a4bac611a4

### CODE REVIEW

**Comment by ide** [comment]
The SDK 33 branch created merge conflicts, sorry about that.

File: android/expoview/src/main/java/host/exp/exponent/AppLoader.java [hunk 1]
```diff
@@ -123,17 +123,8 @@ public void onCompleted(JSONObject manifest) {
...
-            if (ABIVersion.toNumber(manifestSdkVersion) < ABIVersion.toNumber("26.0.0")) {
-              fallbackToCacheTimeout = DEFAULT_TIMEOUT_LENGTH_BEFORE_SDK26;
-            }
-          }
-        } else {
-          // only support checkAutomatically: never in shell & detached apps
-          shouldCheckForUpdate = true;
-        }
+        // only support checkAutomatically: never in shell & detached apps
+        shouldCheckForUpdate = true;
```
Comment (esamelson) [inline android/expoview/src/main/java/host/exp/exponent/AppLoader.java]: this still needs to be wrapped in `if (!Constants.isStandaloneApp())` I believe, unless I'm missing something

**Review by esamelson — CHANGES_REQUESTED** [review]
did not try to look at iOS side, but Android looks good to me other than this one change!

File: android/expoview/src/main/java/host/exp/exponent/AppLoader.java (hunk 1 above)
Comment (sjchmiela) [reply android/expoview/src/main/java/host/exp/exponent/AppLoader.java]: Phew, thanks for catching that!

**Review by tsapeta — APPROVED** [review]

**Pull request merged**